src/main/java/uk/co/bconline/ndelius/service/impl/UserServiceImpl.java

```diff
@@ -248,4 +250,19 @@ public void updateUser(User user)
 		myDatasets.add(userEntryService.getUserHomeArea(myUsername()));
 		return myDatasets;
 	}
+
+
+	private List<SearchResult> expandEmailSearchToDB(SearchResult sr, String query)
+	{
+		List<SearchResult> results = new ArrayList<>();
+
+		// Search the database to obtain staff and team records if the query is a substring of the search result's email
+		if (sr.getSources().contains("LDAP") && sr.getEmail() != null && sr.getEmail().contains(query))
```

Comment:
I think `query` needs to be tokenized before checking for a substring.

For example if I search for multiple terms - `query="marcus marcus.aspin@example.com"` should match on the email part but doesn't on the whole string.